### PR TITLE
Add homepage to website

### PR DESCRIPTION
**Pull request by anamatasanch**
+Add mobile friendly Navbar with placeholder links -> https://drive.google.com/file/d/123InmSDbkinJMDifEAg6xVqy1-qAS4eW/view?usp=sharing
+Add homepage layout:
 -Name and contact containers -> https://drive.google.com/file/d/1Cyi5GMMgcV2yudtbB61eijhVbhbyt5IF/view?usp=sharing
 -Projects container -> https://drive.google.com/file/d/1vrLkYOlO0tfMeKZ3cgPr7MredzJINUp_/view?usp=sharing
 -Experience container -> https://drive.google.com/file/d/1GtSqJqEpUm5x_P_X41mTVV8PFdrD7jln/view?usp=sharing

### CODE REVIEW

File: portfolio/src/main/webapp/js/script.js [hunk 1]
```diff
@@ -25,4 +25,4 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
-}
+}
```
Comment (TsueiCat) [inline portfolio/src/main/webapp/js/script.js:29]: Add the newline at the end of file.

**Review by TsueiCat — CHANGES_REQUESTED** [review]
Could you include a screenshot of this change to the description of your pull request? Just so that it's clear to the reviewer what's being changed here/ can see the ui change if there are any.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,3 @@
...
-
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/google/software-product-sprint.git`
-- Open the GitHub setup walkthrough: `teachme ~/software-product-sprint/walkthroughs/week-0-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+This repo contains Anita's portfolio and SPS projects.
```
Comment (joanapascoal) [inline README.md]: This is also in the previous PR, I think that this files doesn't belong to this PR.

File: portfolio/pom.xml [hunk 3]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>amatasanchez-sps-summer20</deploy.projectId>
```
Comment (joanapascoal) [inline portfolio/pom.xml]: Same about this change file being replicated.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,50 @@
...
+        </script>
+    
+        <h1>Ana Mata's Portfolio</h1>
+        <p>I'm a junior at The University of Texas at El Paso majoring in Computer science.
+          My interests include web development and databases.
+        </p>
+        <img src="Images/ana.jpg" style="height:50vh" alt="Ana Mata">
+        <p>Click here to get a random anime recomendation:</p>
+        <button class="btn waves-effect waves-light" onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>
```
Comment (joanapascoal) [inline portfolio/src/main/webapp/index.html]: This is also in the previous PR. Probably it has a better fit here, so please consider remote it from the previous one.
For that, you can just unstage the file on the PR #1 , command 'git reset path/to/my-file.txt' should work.

**Review by joanapascoal — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (anamatasanch) [reply README.md]: Fixed!

File: portfolio/pom.xml (hunk 3 above)
Comment (anamatasanch) [reply portfolio/pom.xml]: Fixed!

File: portfolio/src/main/webapp/js/script.js (hunk 1 above)
Comment (anamatasanch) [reply portfolio/src/main/webapp/js/script.js:29]: Fixed!

**Review by anamatasanch — COMMENTED** [review]
Extended navbar -> https://drive.google.com/file/d/123InmSDbkinJMDifEAg6xVqy1-qAS4eW/view?usp=sharing
Normal navbar -> https://drive.google.com/file/d/1Cyi5GMMgcV2yudtbB61eijhVbhbyt5IF/view?usp=sharing

**Review by joanapascoal — APPROVED** [review]

File: portfolio/src/main/webapp/js/script.js [hunk 5]
```diff
@@ -17,12 +17,13 @@
...
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
-}
+  
+}
```
Comment (TsueiCat) [inline portfolio/src/main/webapp/js/script.js:29]: I think you add it before the closing bracket, instead of after :) Make sure to change this before merging

**Review by TsueiCat — APPROVED** [review]
